sdk/batch/azure-batch/CHANGELOG.md

```diff
@@ -1,10 +1,6 @@
-.. :changelog:
+# Release History
 
-Release History
-===============
-
-8.0.0 (2019-8-5)
-++++++++++++++++++
+## 8.0.0 (2019-8-5)
```

Comment:
I noticed that the date format between the changelog entries in these files are different. Some are YYYY-M-D and some are YYYY-MM-DD. Would be good to get them consistent whilst you are touching the files anyway.

Reply:
thanks I didn't catch that. I think I'll leave that for library owners to correct. its a giant number of files and every moment this PR stay open means more merge conflicts occurring.